src/README.md

```diff
@@ -1,9 +1,14 @@
+<<<<<<< HEAD
```

Comment:
I committed again, and it should fix this conflict. Previously, the commit did not go through. I guess.